app/static/app/js/components/Map.jsx

```diff
@@ -329,8 +333,21 @@ https://a.tile.openstreetmap.org/{z}/{x}/{y}.png
   }
 
   componentDidUpdate(prevProps) {
+    let opacity = this.props.opacity / 100;
+    
+    Object.values(this.autolayers._layers)
```

Comment:
The probably has to be a better way to do this, maybe by extending the autolayers control, but JS is not really my strengh.